### ISSUE TITLE
Does not support Big Sur.

### ISSUE THREAD

**Issue by Instructor123** [0xbf00/simbple#7]
Discussed in previous [issue](https://github.com/0xbf00/simbple/issues/6), currently does not support Big Sur. When built and run creates the below error.

<img width="1059" alt="simbple_error" src="https://user-images.githubusercontent.com/52050853/137186500-6f8d4963-4fe2-4c97-a79e-30ae1c91e686.png">

**Comment by 0xbf00** [comment]
I'm a little bit confused about the `Container.plist` file you are using there. For me on Big Sur, there are no `Container.plist` files to be found anywhere. Instead, the relevant file seems to have moved to a files with the name `.com.apple.containermanagerd.metadata.plist`. What's up with that?

**Comment by Instructor123** [comment]
Hm that's really interesting; is that still in the ~/Library/Containers folder under different applications? Or is that in a different place?

**Comment by 0xbf00** [comment]
Yep, same folder. Are you sure your `Container.plist` files were created under Big Sur? Maybe you upgraded and these are still from the old system?

**Comment by Instructor123** [comment]
I did upgrade to Big Sur, so it could easily be left over. I'll have to build a VM and check it out.

UPDATE:
Yes it does appear that all of the Container.plist files have disappeared and moved to a hidden file.
